clang/lib/Sema/SemaStmtAttr.cpp

```diff
@@ -207,6 +174,46 @@ Sema::BuildSYCLIntelFPGAIVDepAttr(const AttributeCommonInfo &CI, Expr *Expr1,
       SYCLIntelFPGAIVDepAttr(Context, CI, SafeLenExpr, ArrayExpr, SafelenValue);
 }
 
+template <typename FPGALoopAttrT>
+FPGALoopAttrT *Sema::BuildSYCLIntelFPGALoopAttr(const AttributeCommonInfo &A,
+                                                Expr *E) {
+
+  unsigned SafeInterval = 0;
+  Expr *SafeExpr = nullptr;
+  if (E->isInstantiationDependent()) {
+    SafeExpr = E;
+  } else {
+    llvm::APSInt ArgVal(32);
+
+    if (!E->isIntegerConstantExpr(ArgVal, getASTContext())) {
+      Diag(E->getExprLoc(), diag::err_attribute_argument_type)
+          << (A.getParsedKind() == ParsedAttr::AT_SYCLIntelFPGAMaxConcurrency
+                  ? "'max_concurrency'"
+                  : "'ii'")
+          << AANT_ArgumentIntegerConstant << E->getSourceRange();
+      return nullptr;
+    }
+
+    int Val = ArgVal.getSExtValue();
+
+    if (A.getParsedKind() == ParsedAttr::AT_SYCLIntelFPGAII) {
+      if (Val <= 0) {
+        Diag(E->getExprLoc(), diag::err_attribute_requires_positive_integer)
+            << "'ii'" << /* positive */ 0;
+        return nullptr;
+      }
```

Comment:
Suggest removing braces for the inner condition (the same applies to the succeeding `else` branch)

Reply:
Please don't remove the bracers from the outer condition.